### PR TITLE
Properly use a different eval config

### PR DESCRIPTION
**Pull request by simlmx**
I used this to train on `uk_pv` and test on `mone`. I could not finish the experiments on my last day :(

@zakwatts, @peterdudfield , this is going to be yours to continue :D

### CODE REVIEW

File: psp/scripts/eval_model.py [hunk 1]
```diff
@@ -91,7 +93,7 @@ def main(
     limit,
     split_name,
     log_level,
-    eval_config,
+    eval_config_path,
```
Comment (peterdudfield) [inline psp/scripts/eval_model.py:96]: im sure they have to match exacatly, but it makes it more readable if they do

**Review by peterdudfield — APPROVED** [review]

File: psp/scripts/eval_model.py [hunk 2]
```diff
@@ -41,6 +41,7 @@
 )
 @click.option(
     "--eval-config",
+    "eval_config_path",
```
Comment (zakwatts) [inline psp/scripts/eval_model.py:44]: @peterdudfield The destination of how the value is stored is set to eval_config_path here.

Comment (zakwatts) [reply psp/scripts/eval_model.py:44]: if this wasnt stated then you would need to use eval_config

Comment (peterdudfield) [reply psp/scripts/eval_model.py:44]: ok, sounds good

Comment (simlmx) [reply psp/scripts/eval_model.py:44]: The rationale for using different names was that when calling the script, `--eval-config` is more succinct and clear enough; but once in the code, there is a difference between the path and the config itself. But `--eval-config-path` would work too.

**Pull request merged**